nav2_costmap_2d/plugins/inflation_layer.cpp

```diff
@@ -86,14 +86,17 @@ void InflationLayer::onInitialize()
   }
   matchSize();
 
-  node_->set_parameter_if_not_set("enabled_inflation_layer",true);
-  node_->set_parameter_if_not_set("inflation_radius", 0.55);
-  node_->set_parameter_if_not_set("cost_scaling_factor", 10.0);
-  node_->set_parameter_if_not_set("inflate_unknown",false);
+  node_->set_parameter_if_not_set(name_ + "." + "enabled",true);
```

Comment:
Why not have the dots as part of next strings? Are you trying to emphasize the presence of the dots?